javascript/packages/fury/package.json

```diff
@@ -1,10 +1,11 @@
 {
   "name": "@furyjs/fury",
-  "version": "0.5.6-beta",
+  "version": "0.5.7-beta",
   "description": "A blazing fast multi-language serialization framework powered by jit and zero-copy",
   "main": "dist/index.js",
+  "module": "dist/esm/index.js",
   "scripts": {
-    "build": "tsc",
+    "build": "tsc -p tsconfig.json && tsc -p tsconfig.esm.json",
```

Comment:
this may produce two typings, and we indeed need one.

we can set `declaration: false` in the tsconfig.esm.json, and use:
```
    "declaration": true,
    "declarationDir": "types"
```

in the tsconfig.json.

finally, set `"types": "types/index.d.ts"` in package.json